src/main/java/com/odysseusinc/arachne/datanode/controller/analysis/AnalysisEventController.java

```diff
@@ -0,0 +1,74 @@
+/*
+ *
+ * Copyright 2019 Odysseus Data Services, inc.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * Company: Odysseus Data Services, Inc.
+ * Product Owner/Architecture: Gregory Klebanov
+ * Authors: Pavel Grafkin, Vitaly Koulakov, Anastasiia Klochkova, Sergej Suvorov, Anton Stepanov
+ * Created: Jul 8, 2019
+ *
+ */
+
+package com.odysseusinc.arachne.datanode.controller.analysis;
+
+import com.odysseusinc.arachne.datanode.event.AnalysisResultEvent;
+import com.odysseusinc.arachne.datanode.event.AnalysisUpdateEvent;
+import com.odysseusinc.arachne.datanode.model.analysis.Analysis;
+import com.odysseusinc.arachne.datanode.service.AnalysisService;
+import com.odysseusinc.arachne.execution_engine_common.api.v1.dto.AnalysisExecutionStatusDTO;
+import java.io.File;
+import java.util.Optional;
+import org.springframework.context.event.EventListener;
+import org.springframework.core.convert.support.GenericConversionService;
+
+public class AnalysisEventController {
```

Comment:
Why do we really need it? I mean, these are synchronous events received from the same module